index.js

```diff
@@ -127,6 +127,6 @@ function getUserInfoDefault (req) {
 }
 
 module.exports = fp(csrfPlugin, {
-  fastify: '4.x',
-  name: '@fastify/csrf-protection'
+  fastify: '3.x',
+  name: 'fastify-csrf'
```

Comment:
Is not changing the name expected?

Reply:
Yes, we use the old name for all `fastify@3` scoped package